packages/core/test/staticClient/headers.test.ts

```diff
@@ -258,4 +259,30 @@ describe("Base Client headers", () => {
       expect(nock.isDone()).to.be.true;
     });
   });
+
+  describe("getObjectFromResponse", () => {
+
+    it("should return a map of headers", () => {
+      const headersMap = {
+        authorization: "Bearer token",
+        connection: "close"
+      }
+      const headers = new Headers(headersMap);
+
+      expect(exportHeadersAsMap(headers)).to.deep.equal(headersMap);
+    });
+
+    it("should return a comma separated list if a header has multiple values", () => {
+      const headersMap = {
+        accept: "text/plain, text/html"
+      }
+      const headers: Headers = new Headers(headersMap);
+
+      expect(exportHeadersAsMap(headers)).to.deep.equal(headersMap);
+    });
+
+    it("should return an empty map if no headers are passed", () => {
```

Comment:
```suggestion
    it("returns an empty map if no headers are passed", () => {
```
As per best practices style guide.